drivers/hmis/app/models/hmis/hud/processors/disability_group_processor.rb

```diff
@@ -10,7 +10,19 @@ def process(field, value)
       if field_mapping.key?(field)
         disability_type, disability_field, enum_type = field_mapping[field]
         disability_value = attribute_value_for_enum(enum_type, value)
-        @processor.send(disability_type).assign_attributes(disability_field => disability_value)
+
+        attributes = { disability_field => disability_value }
+
+        if Hmis::Hud::Disability.positive_responses.include?(disability_value.to_i)
```

Comment:
Collecting `indefinite_and_impairs` like this is a change from the intake form's behavior. I'd be fine with reverting it and just collecting all of them as nil. It looks like the warehouse also does some processing around this to return a sensible value even if the database info is incomplete? https://github.com/greenriver/hmis-warehouse/blob/10de915680000021ecdd89df692f4cf3f22a6dd8/app/models/grda_warehouse/hud/disability.rb#L132-L139

Reply:
Yeah I don't think this is necessary, we already have the logic encoded in our frontend disability component

Reply:
Would you like to keep the special case around hiv/aids and developmental disability (not collected for pit but in the generic case) or trust the value coming from frontend now that we know we will ask that question (though not require it)?

Reply:
That logic makes me a little nervous, because it's the DisabilityGroup setting an attribute on a _different_ record (the Enrollment). Afaik we don't have anything like that. 2 thoughts:
1. Is this logic that should actually belong on the Enrollment Processor, since it's about setting an Enrollment field? e.g. EnrollmentProcessor checking for presence of `hud_values[DisabilityGroup.developmentalDisability]` or whatnot and setting Enrollment.disablingCondition based on those
2. Doesn't this logic introduce a dependency between the order of factories? I don't think we iterate through fields in a consistent order. Sometimes it might process Disability fields first, other times it might process Enrollment first.


I'm sure we could find reasonable answers to those issues, but they feel somewhat hairy. I don't think we have a compelling reason to add this complexity/effort. Instead, it could just be a point of training for the users to know when to select "yes" for the disabling condition question. (And on the HMIS side, we already made the choice to encapsulate this logic in `DisabilityTable.tsx`, so the responsibility of generating a correct DisabilityCondition already kinda lives with the form itself).

Reply:
Ok, I'm happy with that! I'll remove this. I added a note to our tchc meeting notes about the training point you mentioned. I don't know if it is exactly the right place, but just didn't want to forget it entirely.

Reply:
Perfect, thank you